src/plugins/chartjs/timeline/timeline-chart.module.ts

```diff
@@ -385,6 +387,138 @@ export class TimelineChartPlugin {
         }
     }
 
+    private handleMouseMove(chart: TimelineChart, event: Partial<MouseEvent>): void {
+        const mousePosition = this.isWithinHandle(chart, event);
+
+        let timelineOptions = chart.options as TimelineChartOptions;
+        let hasTooltipOnRange: boolean = timelineOptions.timeline.range.hasOwnProperty('tooltip');
+        let hasTooltipOnHandles: boolean = timelineOptions.timeline.handles.hasOwnProperty('tooltip');
+        let timelineTooltipText: string;
+        let handleTooltipText: { rangeLower: string, rangeUpper: string };
+
+        if (hasTooltipOnRange) {
+            timelineTooltipText = timelineOptions.timeline.range.tooltip.label();
+        }
+
+        if (hasTooltipOnHandles) {
+            handleTooltipText = timelineOptions.timeline.handles.tooltip.label();
+        }
+
+        if (mousePosition === TimelineHandle.Range && hasTooltipOnRange) {
+            this.externalTooltipHandler(chart, TimelineHandle.Range, timelineTooltipText);
+        } else if (mousePosition === TimelineHandle.Lower && hasTooltipOnHandles) {
+            this.externalTooltipHandler(chart, TimelineHandle.Lower, handleTooltipText.rangeLower);
+        } else if (mousePosition === TimelineHandle.Upper && hasTooltipOnHandles) {
+            this.externalTooltipHandler(chart, TimelineHandle.Upper, handleTooltipText.rangeUpper);
+        } else {
+            let tooltipEl = this.getOrCreateTooltip(chart);
+            tooltipEl.style.opacity = '0';
+        }
+
+    }
+
+
+    private getOrCreateTooltip(chart: TimelineChart) {
+        let tooltipEl = chart.canvas.parentNode.querySelector('.timeline-tooltip') as HTMLElement;
+
+        if (!tooltipEl) {
+            tooltipEl = document.createElement('div');
+            tooltipEl.classList.add('timeline-tooltip');
+            tooltipEl.classList.add('tooltip');
+
+            tooltipEl.style.zIndex = '1';
+            tooltipEl.style.width = '-moz-fit-content';
+            tooltipEl.style.width = 'fit-content';
```

Comment:
Does it need a width at all? I notice that `fit-content` is not supported by IE.

https://developer.mozilla.org/en-US/docs/Web/CSS/width